README.md

```diff
@@ -290,37 +267,27 @@ If you use another Linux distribution please install the corresponding packages.
         make
         sudo make install
 
-#### `configure` options
-
-You can use these optional arguments to `./configure` for creating a Makefile:
+#### Options for `configure`
 
-    --no-fuse-group | --fuse-group (only COMMON)
-        Some distributions require user to be in group 'fuse' to use
-        sshfs and encfs. This toggles the check on or off.
-
-    --python3 | --python (all)
-        Use either 'python3' or 'python' to start Python Version 3.x
-
-Note: The first value is default.
-
-See also `common/configure --help` and `qt/configure --help`
-
-
-## NewsFeed
-
-Back In Time has an RSS feed
-https://feeds.launchpad.net/backintime/announcements.atom
+You can use these optional arguments to `./configure` for creating a Makefile.
+See `common/configure --help` and `qt/configure --help` for details.
 
 ## Contribute
-
-There is a mailing list for development topics:
-https://mail.python.org/mailman3/lists/bit-dev.python.org/
-
-There is a developer documentation on https://backintime-dev.readthedocs.org
-It's not complete yet but I'm working on it. If you'd like to contribute
-please add docstrings following the
-[Google style guide](https://sphinxcontrib-napoleon.readthedocs.org/en/latest/example_google.html)
-and add unit-tests for new methods in common. To run unit-test locally you can
-run `cd common && ./configure && make && make test`
+### Resources
+ - Mailing list [bit-dev](https://mail.python.org/mailman3/lists/bit-dev.python.org/) for development related topics
+ - [Source code documentation for developers](https://backintime-dev.readthedocs.org)
+ - [Translations](https://translations.launchpad.net/backintime) are done on a separate plattform
+### Guidelines & Rules
+The maintaining team will welcome all types of contributions. No contribution will be rejected
+just because it doesn't fit to our quality standards, guidelines or rules. Every contribution
+is reviewed and if needed will be improved together with the maintainers.
+
+If you can and know how it will improve the process and lower the burden of maintain if the
```

Comment:
Many details, I would shorten this e.g. into:

"Please take the following best practices into account if possible (to reduce the work load of the maintainers):"